package.json

```diff
@@ -26,7 +26,7 @@
   },
   "scripts": {
     "test": "xo && npm run prepublish && nyc ava",
-    "prepublish": "babel lib --out-dir=dist --optional=runtime",
+    "prepublish": "babel lib --out-dir=dist --presets es2015 --plugins transform-async-to-generator,transform-runtime",
```

Comment:
Maybe it's cleaner to just add a babel configuration property to `package.json`?

```json
{
  "babel": {
    "plugins": [
      "transform-async-to-generator",
      "transform-runtime"
    ],
    "presets": [
      "es2015"
    ]
  }
}
```